index.html

```diff
@@ -63,8 +64,8 @@ <h3>POWERS</h3>
       </section>
 
       <footer>
-        <p class="ref">All information in this page is derived from the <a href="https://disney.fandom.com/wiki/Thor_Odinson">Disney Fandom</a> website.</p>
-        <a href="https://en.wikipedia.org/wiki/Thor_(film)" id="tribute-link" target="_blank"> Learn More</a>
+        <p class="ref">All information in this page is derived from the <a href="https://disney.fandom.com/wiki/Thor_Odinson" rel=”noopener”>Disney Fandom</a> website.</p>
+        <a href="https://en.wikipedia.org/wiki/Thor_(film)" id="tribute-link" rel=”noopener” target="_blank"> Learn More</a>
```

Comment:
Please add a rel="nonreferrer" attribute to the link elements to fix your lint errors.